web/src/lightbox.ts

```diff
@@ -14,12 +14,12 @@ import * as rows from "./rows.ts";
 import * as util from "./util.ts";
 
 type Payload = {
-    user: string | undefined;
-    title: string | undefined;
-    type: string;
-    preview: string;
-    source: string;
-    url: string;
+    user: string | undefined; // Sender's full name
+    title: string | undefined; // aria-label (or link URL, as fallback) of media
+    type: string; // "image" / "inline-video" / "youtube-video" / "vimeo-video" / "embed-video"
+    preview: string; // URL to use in message list or carousel
+    source: string; // URL to use for display in the lightbox
+    url: string; // URL to the original resource for download
```

Comment:
Our JS codebase generally prefers a style of comments on the line before what they're describing; it's a bit more readable and helps encourage writing descriptions long enough that they might need to wrap.